### ISSUE TITLE
They can work in PyQt5,but can not work in pyside2

### ISSUE THREAD

**Issue by BBlance** [thecodemonkey86/qt_mysql_driver#6]
I put files in the corresponding path of pyqt5 (sqldrivers path, bin path), it is available, but when I put it in the corresponding path of pyside2, there is no QMYSQL in the available drivers. There is no bin path in the path of pyside2, so I directly put libmysql.dll in the root directory of pyside2, and put the files in sqldrivers in the correct location. The python I use is the packaged version of anaconda. So I am confused.

**Comment by thecodemonkey86** [comment]
Hi,
maybe it helps if you set the environment variable QT_DEBUG_PLUGINS=1 in Qt Creator to find out what's wrong

![Unbenannt](https://user-images.githubusercontent.com/11927938/100941002-35db5600-34f9-11eb-8139-2f11ae16c33f.PNG)

the debug output on my machine then gave the hint, that Python is searching for the qsqlmysql.dll in the directory

 C:\Users\ _{user name}_ \AppData\Local\Programs\Python\Python39\sqldrivers

. And I put the libmysql and OpenSSL dll's in an arbitrary directory which is registered in the Windows PATH variable

![image](https://user-images.githubusercontent.com/11927938/100941520-0bd66380-34fa-11eb-8ef5-d4c08a8772a6.png)

hope it helps

**Comment by BBlance** [comment]
Thank you for your reply. According to your method, I did debugging and found that the mysql driver failed to load. Although the reason was found, the driver loading problem was not solved. However, during the most recent debugging, I accidentally replaced the MinGW version of qsqlmysql.dll with the MSVC version, and I tried to run it, and it worked normally. This phenomenon is incomprehensible. MSVC is not installed in my operating system, and I have not even installed any Microsoft programming language.
